src/WarcraftLegacies.Source/PassiveAbilities/SpellConduction/SpellConductionBuff.cs

```diff
@@ -53,8 +53,8 @@ private void OnDamageTaken()
     }
 
     private void DamageCaster(unit damager, float eventDamage) =>
-      Caster.TakeDamage(damager, eventDamage * RedirectionPercentage, false, true, ATTACK_TYPE_CHAOS,
-        DAMAGE_TYPE_UNIVERSAL, BlzGetEventWeaponType());
+      Caster.TakeDamage(damager, eventDamage * RedirectionPercentage, false, true, BlzGetEventAttackType(),
```

Comment:
If you haven't already, I'd recommend updating the tooltip to clearly describe the new behaviour; i.e. that both the victim's AND the Obsidian guy's resistances will apply to the damage.

Reply:
Updated